genome.spec

```diff
@@ -1,34 +1,45 @@
 /*
-KBase genome ID
+Workspace reference to a genome object.
```

Comment:
This isn't a reference, those are `@id ws`

Reply:
If I read https://kbase.us/services/ws/docs/typedobjects.html?highlight=annot#idannotations, I see this annotation being referred to as a "reference" all over the place. Are you using some special definition of "reference" that you like? What would you call this instead?

Reply:
This is a `@id kb`, not a `@id ws`. I should change reference in the `@id kb` documentation to something else as that term is overloaded.

Reply:
https://github.com/kbase/workspace_deluxe/pull/339/commits/73dfbb9f63ce7f442f12ee668ccf4ae79b8733b3

Reply:
Changed the descriptor here based on out-of-band conversation.
Not sure how this is missing from genome objects when it's not listed as optional.
Will ask Jason about it.